python/torch_mlir/tools/import_onnx/__main__.py

```diff
@@ -138,6 +143,11 @@ def parse_arguments(argv=None) -> argparse.Namespace:
         action="store_true",
         help="Disable verification prior to printing",
     )
+    parser.add_argument(
+        "--no-data-prop",
```

Comment:
Typically you want flags to be positive. E.g. `--data-prop` with a value of true, then people can set to false

Reply:
Yeah, I was wondering about that. I just figured if data prop was default it would be easier to pass --no-data-prop than --data-prop=False. I'll make the change.

Reply:
I figure the cleanest option is to use argparse.BooleanOptionalAction, which provides both --data-prop and --no-data-prop with default value of True for data prop. Let me know what you think.